.github/ModuleBuildDurationReport.java

```diff
@@ -39,7 +39,7 @@ public class ModuleBuildDurationReport implements Runnable {
 
   // we will assume the previous module ends on this and a new one begins
   private static final Pattern BUILD_START_PATTERN =
-      Pattern.compile("^" + TIMESTAMP + " \\[INFO\\] Building (.+?) (\\S+-SNAPSHOT).* \\[([0-9]+)/[0-9]+\\]");
+      Pattern.compile("^" + TIMESTAMP + " \\[INFO\\] Building Quarkus - (.+?) (\\S+-SNAPSHOT).* \\[([0-9]+)/[0-9]+\\]");
```

Comment:
I changed that myself because actually not every module starts with `Quarkus - `. Things in `independent-projects/` usually don't so we are missing some modules if you do that.